#!/usr/bin/python3


def raise_exception_msg(message=""):
    """ raises the nameError message """
    raise NameError(message)
